argon2-jvm-nolibs/src/main/java/de/mkammerer/argon2/jna/Argon2_version.java

```diff
@@ -0,0 +1,25 @@
+package de.mkammerer.argon2.jna;
+
+import de.mkammerer.argon2.Argon2Version;
+
+/**
+ * argon2_type type for C interaction.
+ */
+public class Argon2_version extends JnaUint32 {
+
+    /**
+     * Constructor.
+     *
+     * @param version Version.
+     */
+    public Argon2_version(int version) {
+        super(version);
+    }
+
+    /**
+     * Constructor.
+     */
+    public Argon2_version() {
```

Comment:
This ctor is unused.

Reply:
JNA requires a `public no-args constructor`.

Reply:
again what learned, like the germans say ;)